proton/src/config.rs

```diff
@@ -1,29 +1,43 @@
 use serde_json;
 use std::fs::File;
 use std::io::{self, Read, Write};
-use std::path::Path;
 
 #[derive(Serialize, Deserialize)]
 pub struct Config {
-    pub key: String,            // Last auth key used (usually user key)
-    pub vixen_folder: String,   // Points to "Sequence Data" Vixen folder
+    // Last auth key used (usually user key)
+    pub key: String,
+    // Points to "Sequence Data" Vixen folder
+    pub vixen_folder: String,
+    // Points to "vixenconverter/converter.py" from proton-vixen-converter
+    pub vixen_converter_py: String,
+    // An unused DMX channel that can be set for unused Vixen channels
+    pub default_dmx_channel: u16
 }
 
 impl Config {
     // Creates a new, default config
     pub fn default_config() -> Config {
-        return Config::new("user.pub", "~/Dropbox/LightShow/Sequencing/Sequence Data");
+        return Config::new(
+            "user.pub",
+            "/home/lightshow/Dropbox/LightShow/Sequencing/Sequence Data",
```

Comment:
See above

Reply:
The relative path wasn't working; I don't think this was expanding like it would in a shell. Could grab the home directory from Rust. Might do another PR with that change

Reply:
`~` sometimes behaves weird inside of programs.  Basically you have to get the string in the environment variable `$HOME` and concatenate it to the path you seek.  Python has a path combiner I've used in the past as well (i think they also have an `os.home`)

Reply:
Rust has https://doc.rust-lang.org/std/env/fn.home_dir.html